CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+
+1. View new issues using this filter: [https://github.com/neontribe/www/issues](https://github.com/neontribe/www/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+no%3Aproject+).
+1. Decide whether the [Definition of done](#definition-of-done) is ready for action on a given ticket.
+   - if further work is needed then assign a related action in the meeting notes
+1. Take tickets that are ready for action and add to `Top priority issues` or `Backlog` columns based on group instincts of user value.
+   - not all tickets will be high enough priority to **make it**
+   - decisions from previous meetings should not be revisited in the weekly cycle
+1. Document reason for decision on tickets, so that it makes sense later.
+
+From time-to-time we may decide to hold more in depth prioritisation meetings to involve more people and to look at bigger issues.
+
+### Definition of done
+
+These differ for each type of issue:
+
+- [`Bug`](https://english.stackexchange.com/a/40935) has clearly documented steps to reproduce the error _and_ we are sure the error has been eliminated.
+- `Story` the definition of done uses Acceptance Criteria, this can be quite broad, specifically when the ticket is first created. We expect that the Acceptance Criteria will be refined during both prioritisation, and development, so that the final goal is clear and testable.
+- `Research` will depend on the context and may be a simple statement of research aims.
+
+## Getting started on development
+
+Any issues in the `Top priority issues` column are important enough and have been prioritised.
+
+When contributing to this project please do the following:
+
+1. Pick something near the top of the `Top priority issues` column, which you feel you have the time and confidence to tackle.
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+1. Check the [Definition of done](#definition-of-done) for the ticket to see if you understand it.
+   - If you need more information please follow up with the person who spawned/created the task.
+1. Assign yourself and move the ticket to the `In progress` column.
+1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. _n.b._ [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
+1. Work on the feature branch, make small commits which include the _reason_ for each commit's change.
+   - We don't expect you to battle the problem alone, put any queries on the github issue and slack, people will help when they have time.
+1. When you're happy with the changeset/the commits you've made, and you're happy that the ticket passes its [Definition of done](#definition-of-done) create a pull request.
+   _n.b._ Pull requests are the pull requester's responsibility, when it receives the request number
```

Comment:
```suggestion
   - Once you have made a pull request it is your responsibility to check for reviews. See [review process] (#review-process)

```

Reply:
cleared up meaning as well as changed the bullet format - I think this is right @RpprRoger